generators/page/templates/vue/src/test/javascript/spec/app/pages/page.service.spec.ts.ejs

```diff
@@ -0,0 +1,34 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+/* tslint:disable max-line-length */
+import <%= pageName %>Service from '@/pages/<%= pageFolderName %>/<%= pageFolderName %>.service';
+
+describe('Service Tests', () => {
+  describe('<%= pageName %> Service', () => {
```

Comment:
Why use two `describe`? 
I think we can have  `beforeEach` and the simple test in the same `describe`. What do you think?

Reply:
I copied the entity templates for the test, that's why there is an extra describe.